### ISSUE TITLE
Add admin notice when activating Allowed Products that all referrals will be blocked

### ISSUE THREAD

**Issue by gspice** [AffiliateWP/affiliatewp-allowed-products#19]
Many times new users will download and install all available AffiliateWP add-ons for their license level not realizing their functionality.  When they install **Allowed Products**, they do not realize this will block all referrals (creating them in the amount of `$0.00`) unless they specify the product ids in the supplied field in _Affiliates > Settings > Integrations_.

<img width="891" alt="allowed-product-ids" src="https://user-images.githubusercontent.com/3018245/49246595-300e7f80-f3ca-11e8-8520-e4efa9881b38.png">
   

**Requested behavior change:**
I would love to add a nag in the admin area that appears when they activate Allowed Products to alert them that all product referrals will be blocked until they specify the product ids.

**Comment by pippinsplugins** [comment]
I agree. This could easily prevent a _lot_ of tickets.

**Comment by amdrew** [comment]
There should already be a notice that shows when you first install it and have not entered any product IDs: https://github.com/AffiliateWP/affiliatewp-allowed-products/blob/master/includes/functions.php#L159-L175

> There are currently no products configured to generate commissions. Visit the Integrations screen to enter some product IDs under the Allowed Products section

The notice is dismissible, @gspice can you test to see if you see the notice?

**Comment by amdrew** [comment]
Related: https://github.com/AffiliateWP/affiliatewp-allowed-products/issues/1

**Comment by DrewAPicture** [comment]
Maybe we should make it _not_ dismissible since it can have such profound effects on a core AffiliateWP behavior.

**Comment by gspice** [comment]
Well, i feel silly, I didn't check the closed issues.  It _is_ there on another site. I must have dismissed it on my main test site.  Not sure if it's possible to have it check the product id field and show the notice again if it is empty?

<img width="1146" alt="allowed-products-adminn" src="https://user-images.githubusercontent.com/3018245/49261820-90b4b100-f3f8-11e8-84e7-7a2a8eac3490.png">

**Comment by DrewAPicture** [comment]
Note: This seems to already be done, please investigate.

**Comment by alexstandiford** [comment]
Confirmed. A fresh install shows this:
![image](https://user-images.githubusercontent.com/8210827/88962683-c583a600-d274-11ea-84bf-863ef0bd26be.png)

This goes away after an allowed product is specified.

It also _comes back_ as soon as the setting field is empty again.

However, it _does not ever come back_ if you click dismiss.

I think I'm with @DrewAPicture on this, and this notice should _not_ be dismiss-able. I went ahead and created a PR for this. If we don't want to take this route - no problem!

PR #27
